charts/celeborn/values.yaml

```diff
@@ -26,7 +26,7 @@ image:
   pullPolicy: Always
   tag: 0.1.1-6badd20
 
-imagePullSecrets: {}
+imagePullSecrets: []
```

Comment:
Could you also fix `imagePullSecrets` in`tests/kubernetes-it/docker/helm/values.yaml`?

Reply:
Sure, updated